packages/database/lib/schema.ts

```diff
@@ -1,5 +1,71 @@
-import { pgTable, text } from 'drizzle-orm/pg-core';
+import { bigint, index, pgTable, primaryKey, smallint, smallserial, varchar } from 'drizzle-orm/pg-core';
+
+// TODO: table(s) representing order book for each market.
```

Comment:
When doing this. We could do L1 (or as some call it L0) books which would just be top of book which would afford us mid price over time. But I don't know if we need more than like L5 for anything historical. That being said, we can aggregate ALL orders to generate these views.

In the past I've aggregated the like L1,L2,L3,L4,L5 in just JSONB in Postgres for use later if needed. I did do to L10, but I'm not sure if I was just insane.

Reply:
Adding an orders table so we can show order history and add this in the future as well.